docs/source/install/tools.rst

```diff
@@ -10,10 +10,57 @@ This document describes how to install following PDS Registry tools (command-lin
  * **Standalone Harvest** extracts metadata from PDS4 labels and loads extracted metadata into PDS Registry.
  * **Harvest Client** has to be used with Scalable Harvest server components to submit asynchronous jobs
    to the server cluster.
- * **Registry Manager** is a tool to create Registry indices in OpenSearch and to 
+ * **Registry Manager** is a tool to create Registry indices in OpenSearch and to
    manage data and LDDs (data dictionaries).
  * **Supplementer** can be used to load supplemental metadata into PDS Registry.
 
+Prerequisites
+*************
+
+Java is required to run many Registry components and tools such as Harvest, Registry Manager,
+and API Server.
+
+Only **Java 11** is supported.
+Some applications, like Harvest or Registry Manager will not run with **Java 1.8**.
+
+
+Test if Java is already installed
```

Comment:
[Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Test If Java Is Already Installed**

This was a tough one. I used the https://titlecaseconverter.com/ get the correct suggestion :-)

Reply:
I need to learn that, `to` stays 'to' but 'if' becomes 'If' as well as 'Is'. It is a whole field of expertise that I had no idea of  !

Reply:
Well, done